assets/aws/files/bin/teleport-generate-config

```diff
@@ -255,6 +255,11 @@ teleport:
   ca_pin: CA_PIN_HASH_PLACEHOLDER
   nodename: ${LOCAL_HOSTNAME}
   advertise_ip: ${LOCAL_IP}
+  cache:
+    type: in-memory
+  connection_limits:
+    max_connections: 6500
```

Comment:
This should be 10x as per https://community.goteleport.com/t/memory-cache-for-proxy/882

```suggestion
    max_connections: 65000
```